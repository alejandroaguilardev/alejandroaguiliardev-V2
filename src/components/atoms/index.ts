export * from "./CardContent/CardContent";
export * from "./Copyright/Copyright";
export * from "./Curriculum/Curriculum";
export * from "./ListOption/ListOption";
export * from "./Logotype/Logotype";
export * from "./Hero/Hero";
export * from "./Networks/Networks";
export * from "./PageRoute/PageRoute";
export * from "./TitleSection/TitleSection";
