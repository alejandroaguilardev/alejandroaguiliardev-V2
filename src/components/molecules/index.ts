export * from './CardItem/CardItem'
export * from './Menu/Menu'
export * from './MenuResponsive/MenuResponsive'
export * from './SocialNetworks/SocialNetworks'