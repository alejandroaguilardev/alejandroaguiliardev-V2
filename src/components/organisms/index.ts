export * from "./About/About";
export * from "./ContainerSocialNetworks/ContainerSocialNetworks";
export * from "./Education/Education";
export * from "./Experience/Experience";
export * from "./Footer/Footer";
export * from "./Navbar/Navbar";
export * from "./Projects/Projects";
export * from "./Skills/Skills";
