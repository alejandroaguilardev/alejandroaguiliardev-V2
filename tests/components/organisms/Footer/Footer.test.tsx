import { render, screen } from "@testing-library/react";
import { Footer } from "../../../../src/components/organisms";

describe("Footer Test", () => {
	test("toMatchSnapshot", () => {
		const { container } = render(<Footer />);

		expect(container).toMatchSnapshot();
		expect(screen.getByText("ALEJANDRO AGUILAR")).toBeTruthy();
		expect(screen.getByText("ALEJANDRO AGUILAR")).toBeTruthy();
	});
});
