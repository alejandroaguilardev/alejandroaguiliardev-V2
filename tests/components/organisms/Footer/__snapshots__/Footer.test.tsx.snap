// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test toMatchSnapshot 1`] = `
<div>
  <div
    class="MuiGrid-root css-1h337u4-MuiGrid-root"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-10 css-1hygxpp-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-6 css-1k7sk4d-MuiGrid-root"
        >
          <a
            href="/"
          >
            <div
              class="MuiGrid-root css-2t8riz-MuiGrid-root"
            >
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2750%27%20height=%2750%27/%3e"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt="Logo Alejandro Aguilar"
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
              <p
                class="MuiTypography-root MuiTypography-body1 MuiTypography-noWrap css-1txd3cf-MuiTypography-root"
              >
                ALEJANDRO AGUILAR
              </p>
            </div>
          </a>
          <p
            class="MuiTypography-root MuiTypography-body1 css-60j085-MuiTypography-root"
          >
            Systems Engineer and software developer, passionate about technology and open to learning new technologies.
          </p>
          <ul
            class="MuiList-root MuiList-padding MuiList-dense css-15hv90u-MuiList-root"
          >
            <li
              class="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-yt5ric-MuiListItem-root"
            >
              <a
                href="https://twitter.com/alejandroagdev/"
                rel="noreferrer noopener"
                target="_blank"
              >
                <button
                  aria-label="Twitter"
                  class="MuiButtonBase-root MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary css-1dm1o0z-MuiButtonBase-root-MuiFab-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1kqeszy-MuiSvgIcon-root"
                    data-testid="TwitterIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </li>
            <li
              class="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-yt5ric-MuiListItem-root"
            >
              <a
                href="https://www.instagram.com/alejandroaguilar.dev/"
                rel="noreferrer noopener"
                target="_blank"
              >
                <button
                  aria-label="Instagram"
                  class="MuiButtonBase-root MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary css-1dm1o0z-MuiButtonBase-root-MuiFab-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1kqeszy-MuiSvgIcon-root"
                    data-testid="InstagramIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M7.8 2h8.4C19.4 2 22 4.6 22 7.8v8.4a5.8 5.8 0 0 1-5.8 5.8H7.8C4.6 22 2 19.4 2 16.2V7.8A5.8 5.8 0 0 1 7.8 2m-.2 2A3.6 3.6 0 0 0 4 7.6v8.8C4 18.39 5.61 20 7.6 20h8.8a3.6 3.6 0 0 0 3.6-3.6V7.6C20 5.61 18.39 4 16.4 4H7.6m9.65 1.5a1.25 1.25 0 0 1 1.25 1.25A1.25 1.25 0 0 1 17.25 8 1.25 1.25 0 0 1 16 6.75a1.25 1.25 0 0 1 1.25-1.25M12 7a5 5 0 0 1 5 5 5 5 0 0 1-5 5 5 5 0 0 1-5-5 5 5 0 0 1 5-5m0 2a3 3 0 0 0-3 3 3 3 0 0 0 3 3 3 3 0 0 0 3-3 3 3 0 0 0-3-3z"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </li>
            <li
              class="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-yt5ric-MuiListItem-root"
            >
              <a
                href="https://www.linkedin.com/in/alejandro-aguilar-879402153/"
                rel="noreferrer noopener"
                target="_blank"
              >
                <button
                  aria-label="LinkedIn"
                  class="MuiButtonBase-root MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary css-1dm1o0z-MuiButtonBase-root-MuiFab-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1kqeszy-MuiSvgIcon-root"
                    data-testid="LinkedInIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </li>
            <li
              class="MuiListItem-root MuiListItem-dense MuiListItem-gutters MuiListItem-padding css-yt5ric-MuiListItem-root"
            >
              <a
                href="https://github.com/alejandroaguilardev/"
                rel="noreferrer noopener"
                target="_blank"
              >
                <button
                  aria-label="GitHub"
                  class="MuiButtonBase-root MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary MuiFab-root MuiFab-circular MuiFab-sizeSmall MuiFab-primary css-1dm1o0z-MuiButtonBase-root-MuiFab-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1kqeszy-MuiSvgIcon-root"
                    data-testid="GitHubIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </a>
            </li>
          </ul>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 css-ot7hnh-MuiGrid-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1469zy3-MuiTypography-root"
          >
            Information
          </p>
          <nav
            aria-label="mailbox folders"
            class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
          >
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    E-mail
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  >
                    hello@alejandroaguilar.dev
                  </p>
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Security Politics
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Terms and Conditions
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
          </nav>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-3 css-ot7hnh-MuiGrid-root"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-1469zy3-MuiTypography-root"
          >
            Briefcase
          </p>
          <nav
            aria-label="mailbox folders"
            class="MuiList-root MuiList-padding css-h4y409-MuiList-root"
          >
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Design
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Front End
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Back End
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button css-bshv44-MuiButtonBase-root-MuiListItem-root"
                role="button"
                tabindex="0"
              >
                <div
                  class="MuiListItemText-root MuiListItemText-multiline css-1itc3p-MuiListItemText-root"
                >
                  <span
                    class="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Smart Contract
                  </span>
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-qybatc-MuiTypography-root"
                  />
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </div>
              <hr
                class="MuiDivider-root MuiDivider-fullWidth MuiDivider-light css-xetjfm-MuiDivider-root"
              />
            </div>
          </nav>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root css-815iv4-MuiGrid-root"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-h3yix-MuiTypography-root"
        >
          © 2020 - 
          2022
           All rights reserved  | Alejandro Aguilar Systems Engineer | Software Developer
        </p>
      </div>
    </div>
  </div>
</div>
`;
